tests/unit/services/ajax-test.js

```diff
@@ -134,6 +134,49 @@ test("request() promise label is correct", function(assert) {
   assert.equal(postPromise._label, 'ember-ajax: POST to /posts');
 });
 
+test("post() promise label is correct", function(assert) {
+  service = Service.create();
+  const url = '/posts';
```

Comment:
That's an example of something that, in any other language, would be considered immutable. Despite you not making a specific reference to immutability, you are still asserting that `const` should only be used in this type of scenario. Thus, I am confused why the ES6 spec bothered to introduce a keyword that is apparently *supposed* to convey this concept while specifically calling for a different implementation altogether. That seems relatively useless, does it not?